components/elm/src/main/elmfates_interfaceMod.F90

```diff
@@ -2221,6 +2233,39 @@ end subroutine wrap_accumulatefluxes
 
  ! ======================================================================================
 
+ subroutine wrap_WoodProducts(this, bounds_clump, fc, filterc)
+
+   ! !ARGUMENTS:
+   class(hlm_fates_interface_type), intent(inout) :: this
+   type(bounds_type)              , intent(in)    :: bounds_clump
+   integer                        , intent(in)    :: fc                   ! size of column filter
+   integer                        , intent(in)    :: filterc(fc)          ! column filter
+   
+   ! Locacs
+   integer                                        :: s,c,icc
+   integer                                        :: nc
+
+   associate(&
+         hrv_deadstemc_to_prod10c     => col_cf%hrv_deadstemc_to_prod10c    , &
+         hrv_deadstemc_to_prod100c    => col_cf%hrv_deadstemc_to_prod100c)
+ 
+    nc = bounds_clump%clump_index
+    ! Loop over columns
+    do icc = 1,fc
+       c = filterc(icc)
+       s = this%f2hmap(nc)%hsites(c)
+
+       ! Shijie: Pass harvested wood products to ELM variable
+       hrv_deadstemc_to_prod10c(c)  = this%fates(nc)%bc_out(s)%hrv_deadstemc_to_prod10c
```

Comment:
I don't think this variable is available in fates API 21 (which E3SM currently points to).  

Here is the FATES commit where it comes in, which is API 24.

https://github.com/NGEET/fates/commit/b0704f3c23cf2ab5c1e639e78b1d6da348d70271#diff-086baf7fb45f7dd5d1708dbac06ccca866d97c9cdacf16e8d2ba137afe1040eeR678 

@glemieux, since I'm currently working on the API 24 PR, should we schedule this PR to come after?  We could combine too.

Reply:
@rgknox yep, this should definitely come in after API 24 update.  I'm cool with combining the two if you'd like.

Reply:
https://github.com/E3SM-Project/E3SM/pull/5092